src/JWT/JWT.cs

```diff
@@ -264,7 +264,12 @@ public static void Verify(string payloadJson, string decodedCrypto, string decod
             var secondsSinceEpoch = Math.Round((DateTime.UtcNow - UnixEpoch).TotalSeconds);
             if (secondsSinceEpoch >= expInt)
             {
-                throw new TokenExpiredException("Token has expired.");
+               var tokenExpiredException = new TokenExpiredException("Token has expired.")
```

Comment:
Cool! Can you please format the document (usually Ctrl+K,D) to make sure no extra/missing spaces were added/removed?